interface-definitions/service_webproxy.xml.in

```diff
@@ -488,7 +488,7 @@
                         <description>Name of source group</description>
                       </valueHelp>
                       <constraint>
-                        <regex>^[^0-9]</regex>
+                        <regex>[^0-9][a-zA-Z_][a-zA-Z0-9][\w\-\.]*</regex>
```

Comment:
I have to admit I don't like negative error messages. "Cannot start with a number" isn't actionable because it doesn't tell me what it _can_ start with. As a user I'd prefer "source-group name must start with...".

I also have to admit I can't remember squidguard rules for names. Can they start with a hyphen or an underscore? Your regex does allow it to start with any non-numeric character, like `_-my_group`.

Reply:
I updated the regex and constraintErrorMessage